### PR TITLE
MM-40659: Skipped tasks incorrectly marked as unfinished

### PR DESCRIPTION
**Pull request by Phrynobatrachus**
#### Summary
When the confirmation for a run finish is shown, skipped tasks are no longer considered "outstanding".

Screenshot from 2022-01-04 16-59-55

Screenshot from 2022-01-04 17-00-03

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-40659

#### Checklist

- ~~Telemetry updated~~
-  ~~Gated by experimental feature flag~~
- ~~Unit tests updated~~

### CODE REVIEW

File: server/app/playbook_run_service.go [hunk 1]
```diff
@@ -780,7 +780,7 @@ func (s *PlaybookRunServiceImpl) OpenFinishPlaybookRunDialog(playbookRunID, trig
 	numOutstanding := 0
 	for _, c := range currentPlaybookRun.Checklists {
 		for _, item := range c.Items {
-			if item.State != ChecklistItemStateClosed {
+			if item.State == "" {
```
Comment (lieut-data) [inline server/app/playbook_run_service.go:783]: Would it make sense to do the same check as in the frontend against `ChecklistItemStateOpen` and `ChecklistItemStateInProgress`?

**Review by lieut-data — CHANGES_REQUESTED** [review]
Thanks for jumping in, @Phrynobatrachus! Just one comment below:

**Review by crspeller — APPROVED** [review]

**Comment by Phrynobatrachus** [comment]
Double checking one more possibly relevant spot in TS.

**Review by lieut-data — APPROVED** [review]
Sweet, thanks!

**Pull request merged**